package.json

```diff
@@ -154,6 +154,7 @@
     "chart.js": "^2.7.3",
     "datatables.net": "^1.10.19",
     "date-fns": "^1.30.1",
+    "date-fns-2.x": "npm:date-fns@2.4.1",
```

Comment:
Nit: Personally, I'd go the other way around (having `date-fns` be v2 and alias v1 to `date-fns-1.x`).

Reply:
I did consider that, but there is another package that uses 1.x so that version would be installed anyway. Therefore it was simpler to do it this way round.